lms/templates/courseware/courses.html

```diff
@@ -21,8 +22,8 @@
     DiscoveryFactory(
       ${course_discovery_meanings | n, dump_js_escaped_json},
       getParameterByName('search_query'),
-      "${user_language}",
-      "${user_timezone}"
+      "${user_language | n, dump_js_escaped_json}",
```

Comment:
Use `js_escaped_string` when inside a string (here and the next line), and use `dump_js_escaped_json` when not inside a string.